### ISSUE TITLE
Increase disc space requirements and add a check in update.sh

### ISSUE THREAD

**Issue by ClundXIII** [mailcow/mailcow-dockerized#2236]
**Is your feature request related to a problem? Please describe.**
I recently ran out of space with 10 GB HDD during an upgrade (the mail directory is on another virtual disc).

**Describe the solution you'd like**
Increase the minimal requried space in Documentation to at least 10 GB. Also add a small warning in update.sh that lets the user abort the update if there is not enough space left. I ran out of space during an upgrade and nearly got an hearth attack since I thought my mail server was messed up. (I do have backups though ;-) )

**Additional context**
I deleted the old volumes every time, so that should not be the issue.

**Comment by andryyy** [comment]
Did you try `docker system prune`? **Only run this commend while mailcow is up and running**, otherwise it will purge your volumes. :(

**Comment by ClundXIII** [comment]
Yes I did. Now I am around 7.something GB. But the problem is that the memory usage spikes during upgrade which is not negligible.

**Comment by ClundXIII** [comment]
I should add that that I also run roundcube as a webmailer.

**Comment by Adorfer** [comment]
I would recommend the update-script before performaning update 
to check for potential low discspace and in case of doubt to ask the operator if it shoud prune unused images. 
(Not only AFTER the update, but as well in beforehand, before even stopping the running containers)

**Comment by andryyy** [comment]
Yes, a low disk space check is a good idea.

**Comment by ClundXIII** [comment]
Is this planned or already implemented?

**Comment by andryyy** [comment]
Planned.
